skin.json

```diff
@@ -146,6 +146,9 @@
       "height": 81
     }
   },
+  "live": {
+    "dvrEnabled": false
```

Comment:
DVR is complicated specifically for HLS - HLS technically always has "DVR" for all live streams (like, minimum 1 chunk of stream). Some customers may want to hide the "DVR" scrubber, and act more like a true live stream.

having a "forceDvrDisabled" is more like "hideScrubberOnLiveStream".  I would be happy to talk about the desired effect here, but I feel like Web will have a similar issue.  I know that HDS or other formats tell you if DVR is part of the stream, but HLS won't